x-pack/plugins/security_solution/public/resolver/view/resolver_loading_state.test.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Simulator } from '../test_utilities/simulator';
+import { pausableNoAncestorsTwoChildren } from '../data_access_layer/mocks/pausable_no_ancestors_two_children';
+import { emptiableNoAncestorsTwoChildren } from '../data_access_layer/mocks/emptiable_no_ancestors_two_children';
+import '../test_utilities/extend_jest';
+
+describe('Resolver: data loading and resolution states', () => {
+  let simulator: Simulator;
+  const resolverComponentInstanceID = 'resolver-loading-resolution-states';
+
+  describe('When entities data is being requested', () => {
+    beforeEach(() => {
+      const {
+        metadata: { databaseDocumentID },
+        dataAccessLayer,
+        pause,
+      } = pausableNoAncestorsTwoChildren();
+      pause({ entities: true });
+      simulator = new Simulator({
+        dataAccessLayer,
+        databaseDocumentID,
+        resolverComponentInstanceID,
+      });
+    });
+
+    it('should display a loading state', async () => {
+      // Trigger a loading state by requesting data based on a new DocumentID.
+      // There really is no way to do this in the view besides changing the url, so triggering the action instead
```

Comment:
:grey_question: Where is the action mentioned in this comment being triggered?

Reply:
errant comment from an earlier implementation. Gonna remove, thanks!